src/Nethermind/Nethermind.Api/NethermindApi.cs

```diff
@@ -75,12 +76,7 @@ public IBlockchainBridge CreateBlockchainBridge()
             LazyInitializer.EnsureInitialized(ref _readOnlyDbProvider, () => new ReadOnlyDbProvider(DbProvider, false));
 
             // TODO: reuse the same trie cache here
-            ReadOnlyTxProcessingEnv readOnlyTxProcessingEnv = new(
-                _readOnlyDbProvider,
-                ReadOnlyTrieStore,
-                readOnlyTree,
-                SpecProvider,
-                LogManager);
+            IReadOnlyTxProcessorSourceExt readOnlyTxProcessingEnv = new ReadOnlyTxProcessingEnv(_readOnlyDbProvider, ReadOnlyTrieStore, readOnlyTree, SpecProvider, LogManager);
```

Comment:
NP: Prefer to put long params in multiple line like before. Keep code to the left and in the same column, easier to read. Also, any addition/remove/change to a parameter will appear as a single line in diff viewer.